rspace/src/test/scala/coop/rchain/rspace/nextgenrspace/history/HistoryGenerativeSpec.scala

```diff
@@ -76,6 +76,32 @@ class HistoryGenerativeSpec
     finalSimplisticHistory.runSyncUnsafe(20.seconds).root shouldBe emptyRootHash
   }
 
+  "history" should "accept new leafs in bulk" in forAll(
```

Comment:
Of course "history", because it's a `History...Spec`. It would be more informative if you wrote which method you are testing